src/Tasks/Drush/Drush.php

```diff
@@ -24,10 +22,11 @@ class Drush extends Exec
     protected $accountMail = '';
     protected $accountName = '';
     protected $accountPassword = '';
-    protected $databaseUser = '';
-    protected $databasePassword = '';
+    protected $databaseType = '';
```

Comment:
Instead of `$databaseType` I would call this `$databaseScheme`. This is more correct.